### DISCUSSION TITLE
Does bevy support mesh color id pass?

### DISCUSSION THREAD

**Discussion by ghost** [bevyengine/bevy#8142] [Q&A]
Does someone know if bevy supports mesh color id pass?

Also it is possible to get the color of a 3d object with mouse picking?

**Comment by nicopap** [comment]
I assume by "mesh color id pass" you mean some sort of buffer that contains individual colors for individual meshes. 

To this question the answer is "no", someone opened a PR but I don't think it went to the end of things https://github.com/bevyengine/bevy/pull/6991

**Reply by ghost** [reply]
Yes, exactly
